app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -114,6 +153,24 @@ def add_state_id_type(id_type, document)
           end
         end
 
+        def add_sex_code(sex_value, document)
+          sex_code = case sex_value
+                     when 'male'
+                       1
+                     when 'female'
+                       2
+                     end
```

Comment:
D'oh! I am withdrawing this concern because I was mistaken about how this works. 🤦‍♂️ 

If a user has a license that does not indicate that they are male or female (whether the user has an "X" or the data is simply not printed on their ID at all), the conditional below will not run, and we just don't `add_optional_element` and all is well.